app/src/main/java/com/github/se/eventradar/ui/component/ViewProfileReusableComponents.kt

```diff
@@ -0,0 +1,48 @@
+package com.github.se.eventradar.ui.component
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.github.se.eventradar.R
+
+@Composable
+fun UserProfileImage(profilePicLink: String, name: String, modifier: Modifier) {
```

Comment:
Nit: There is a similar function in ReusableComponents.kt. No code duplications on CI so not a priority to change.

Reply:
There is a similar one, but the size and shape was different so I decided to create this one :)

Reply:
Gotcha!